kernel/realsense/d4xx.c

```diff
@@ -5639,4 +5628,4 @@ MODULE_AUTHOR("Guennadi Liakhovetski <guennadi.liakhovetski@intel.com>,\n\
 				Shikun Ding <shikun.ding@intel.com>");
 MODULE_AUTHOR("Dmitry Perchanov <dmitry.perchanov@intel.com>");
 MODULE_LICENSE("GPL v2");
-MODULE_VERSION("1.0.1.20");
+MODULE_VERSION("1.0.1.21");
```

Comment:
Please bump to 22 as 5.16 RC is 21
https://github.com/IntelRealSense/realsense_mipi_platform_driver/blob/db6472e0590596a33c37bca9834ca37833aceefb/kernel/realsense/d4xx.c#L5642

Reply:
Please remove this change as the version was already updated

Reply:
squash and drop unnecessary commits.